ryanair/ryanair.py

```diff
@@ -142,25 +142,38 @@ def get_cheapest_return_flights(
         else:
             return []
 
-    @staticmethod
-    def _get_backoff_type():
+    def get_backoff_type():
         if "unittest" in sys.modules.keys():
             return backoff.constant(interval=0)
+        else:
+            return backoff.expo
 
-        return backoff.expo()
-
-    @staticmethod
+    
     def _on_query_error(e):
         logger.exception(f"Gave up retrying query, last exception was {e}")
 
+    @staticmethod
+    def _call_handlers(hdlrs, target, args, kwargs, tries, elapsed, **extra):
```

Comment:
Hey, thanks for the PR, could you fill me in on the purpose of this `_call_handlers` method please?

Reply:
Hi cohaolain, I think it's something that co-pilot decided to add when I was writing the commit lol, it can be removed.